src/components/CurrencyLogo/index.tsx

```diff
@@ -25,6 +28,19 @@ const StyledLogo = styled(Logo)<{ size: string }>`
   background-color: ${({ theme }) => theme.white};
 `
 
+function chainIdToNetwork(networkId: SupportedChainId): Network {
+  switch (networkId) {
+    case SupportedChainId.MAINNET:
+      return 'ethereum'
+    case SupportedChainId.ARBITRUM_ONE:
+      return 'arbitrum'
+    case SupportedChainId.OPTIMISM:
+      return 'optimism'
+    default:
+      return 'ethereum'
```

Comment:
tempted to leave this as `undefined` given we default to `ethereum` in `getTokenLogoURL`.